guides/common/modules/proc_remediating-compliance-failures.adoc

```diff
@@ -13,21 +14,27 @@ Remediation might render the system non-functional.
 * Your user account has a role assigned that has the `view_arf_reports` and `view_hosts` permissions.
 
 .Procedure
-To examine a simplified compliance policy report:
+To examine a simplified compliance policy report and run the compliance failure remediation wizard:
 
 . In the {ProjectWebUI}, navigate to *Hosts* > *Compliance* > *Reports*.
 . In the *Reported At* column, click the time link of the report you want to examine.
-{Project} displays a simplified list of policy rules with the results of the scan.
+{Project} displays a list of log messages describing the results of the scan.
+. Locate a log message that describes a failed compliance check.
+In the *Actions* column, select *Remediation* to open the compliance remediation wizard.
+Follow the wizard to remediate the compliance failure.
++
+The remediation wizard might not available for all compliance failures.
```

Comment:
@ofedoren Is it possible to summarize for which failures the wizard/snippets are _not_ available?

Reply:
Well, if the report doesn't contain any suggested snippets (fixes) for a rule or if the rule result is not failed then the `Remediation` button is not shown, thus the wizard is not available.

Regarding "why there are no snippets for this particular rule?", I'm afraid I don't have answer :/ I'd say it depends on a rule, i.e. it's more about "wrong idea"/approach or the fix fully depends on a user or the fix simply cannot be automated (like there no flags, packages, config files that can be changed to fix the issue). Not sure though :/

```suggestion
The remediation wizard might not be available for all compliance failures.
```

Reply:
@maximiliankolb You also raised this issue in a previous PR. Based on Oleh's answer, I propose to leave line 26 as is. Or can you think of a different wording that wouldn't raise eyebrows like this one does? :)